src/uikit.cs

```diff
@@ -2063,6 +2063,101 @@ interface UIObjectRestoration {
 #endif
 	}
 
+	interface IUIViewAnimating {}
+
+	[iOS(10,0)]
+	[Protocol]
+	[BaseType (typeof(NSObject))]
+	interface UIViewAnimating
+	{
+		[Abstract]
+		[Export ("state")]
+		UIViewAnimatingState State { get; }
+
+		[Abstract]
+		[Export ("running")]
+		bool Running { [Bind ("isRunning")] get; }
+
+		[Abstract]
+		[Export ("reversed")]
+		bool Reversed { [Bind ("isReversed")] get; set; }
+
+		[Abstract]
+		[Export ("fractionComplete")]
+		nfloat FractionComplete { get; set; }
+
+		[Abstract]
+		[Export ("startAnimation")]
+		void StartAnimation ();
+
+		[Abstract]
+		[Export ("pauseAnimation")]
+		void PauseAnimation ();
+
+		[Abstract]
+		[Export ("stopAnimation:")]
+		void StopAnimation (bool withoutFinishing);
+
+		[Abstract]
+		[Export ("finishAnimationAtPosition:")]
+		void FinishAnimationAtPosition (UIViewAnimatingPosition finalPosition);
+	}
+
+	interface IUIViewImplicitlyAnimating {}
+	[iOS(10,0)]
+	[Protocol, Model]
+	interface UIViewImplicitlyAnimating : UIViewAnimating
+	{
+		[Export ("addAnimations:delayFactor:")]
+		void AddAnimations (Action animation, nfloat delayFactor);
+	
+		[Export ("addAnimations:")]
+		void AddAnimations (Action animation);
+	
+		[Export ("addCompletion:")]
+		void AddCompletion (Action<UIViewAnimatingPosition> completion);
+	
+		[Export ("continueAnimationWithTimingParameters:durationFactor:")]
+		void ContinueAnimationWithTimingParameters ([NullAllowed] IUITimingCurveProvider parameters, nfloat durationFactor);
```

Comment:
Should be `ContinueAnimationWithTimingParameters` just like it's implemented below (in `UIViewPropertyAnimator`)